addons/api/addon/generated/models/managed-group.js

```diff
@@ -0,0 +1,54 @@
+import BaseModel from '../../models/base';
+import { attr } from '@ember-data/model';
+
+/**
+ * A managed group is a resource that represents a collection of accounts.
+ */
+export default class GeneratedManagedGroupModel extends BaseModel {
+  // =attributes
+
+  @attr('string', {
+    description: 'Optional name for identification purposes.',
+  })
+  name;
+
+  @attr('string', {
+    description: 'Optional user-set description for identification purposes.',
+  })
+  description;
+
+  @attr('date', {
+    description: 'The time this resource was created',
+    readOnly: true,
+  })
+  created_time;
+
+  @attr('date', {
+    description: 'The time this resource was last updated.',
+    readOnly: true,
+  })
+  updated_time;
+
+  @attr('number', {
+    description: 'Current version number of this resource.',
+  })
+  version;
+
+  @attr('string', {
+    description: 'The type of this ManagedGroup.',
+  })
+  type;
+
+  @attr('string', {
+    description:
+      'The ID of the Auth Method that is associated with this ManagedGroup',
+  })
+  auth_method_id;
+
+  @attr('array', {
```

Comment:
@randallmorey  You are right.
I made the change but did not understand why `member_ids` `readOnly` field is treated differently. Is it because its type is an array?

Reply:
It doesn't have a type actually.  Which means it can take any value.  Is this field expressed in the protobufs?

Reply:
Yes, it is [here](https://github.com/hashicorp/boundary/blob/main/internal/proto/local/controller/api/resources/managedgroups/v1/managed_group.proto#L47) but now you mention, it has the `repeated` before the type definition.

Reply:
I assume "repeated string" means `String[]`.  Let's keep it in the generated file for now.  However, there should be no type defined as there isn't an `array` transform.